eslint.config.mjs

```diff
@@ -391,7 +395,14 @@ export default [
       '@typescript-eslint/no-unsafe-member-access': 'off',
       '@typescript-eslint/no-unsafe-call': 'off',
       '@typescript-eslint/no-unsafe-assignment': 'warn',
-      '@typescript-eslint/no-unused-expressions': 'warn',
+      '@typescript-eslint/no-unused-expressions': [
+        'warn',
+        {
```

Comment:
why?  https://eslint.org/docs/latest/rules/no-unused-expressions#options

defaults are false for these, right?